l10n/en/products/vpn/download.ftl

```diff
@@ -10,24 +10,45 @@ vpn-download-page-description = Download { -brand-name-mozilla-vpn } for { -bran
 # Variables:
 #   $devices (number) - maximum number of connected devices
 vpn-download-sub-heading = You can download { -brand-name-mozilla-vpn } on up to { $devices } devices with one subscription.
+vpn-download-mozilla-vpn-offers = { -brand-name-mozilla-vpn } offers safe and secure protection for all major operating systems.
 
 vpn-download-not-available-in-country = Sorry, { -brand-name-mozilla-vpn } is not available for download in your country.
 vpn-download-button-copy = Download
+
+# Obsolete string
 vpn-download-for-windows = VPN for { -brand-name-windows } 10/11
+
+vpn-download-for-windows-v2 = VPN for { -brand-name-windows }
 vpn-download-for-windows-requirements = { -brand-name-windows } 10/11 (64-bit only)
+vpn-download-download-for-windows = Download { -brand-name-mozilla-vpn } for { -brand-name-windows }
```

Comment:
By no means a big issue, but I noticed in reviewing that the string IDs can get a little confusing here in places?

We have things like:

- `vpn-download-for-windows`
- `vpn-download-download-for-windows`

Might something like adding `-long` as a suffix instead be a little clearer perhaps? So we'd have both the short and long string equivalents?

- `vpn-download-for-windows`
- `vpn-download-for-windows-long`

Up to you though, not a blocker 😉

Reply:
🧠 🌠